spec/unit/shoulda/matchers/active_model/validate_numericality_of_matcher_spec.rb

```diff
@@ -1141,6 +1216,21 @@ def define_model_validating_numericality(options = {})
     end
   end
 
+  def define_model_validating_numericality_of_virtual_attribute(options = {})
+    virtual_attr = options.delete(:virtual_attribute) do
+      virtual_attribute_name
+    end
+
+    define_model 'Example' do |model|
```

Comment:
`define_model` creates an ActiveRecord model. We also have a `define_active_model` helper you could use that defines an ActiveModel::Model model instead. That means you could drop the next line actually. So it would look like:

``` ruby
define_active_model('Example', accessors: [virtual_attr]) do |model|
```